cypress/features/regression/table.feature

```diff
@@ -3,28 +3,28 @@ Feature: Table component
 
   @positive
   Scenario: Verify the pagination is visible
-    When I open default "Table" component in noIFrame with "table" json from "commonComponents" using "paginate" object name
+    When I open default "Table Test" component in noIFrame with "table" json from "commonComponents" using "paginate" object name
     Then pagination is visible
 
   @positive
   Scenario: I enable selectable
-    When I open default "Table" component in noIFrame with "table" json from "commonComponents" using "selectable" object name
+    When I open default "Table Test" component in noIFrame with "table" json from "commonComponents" using "selectable" object name
     Then rows are selectable
 
   @positive
   Scenario: I disable selectable
-    When I open default "Table" component in noIFrame with "table" json from "commonComponents" using "selectableFalse" object name
+    When I open default "Table Test" component in noIFrame with "table" json from "commonComponents" using "selectableFalse" object name
     Then rows are not selectable
 
   @positive
   Scenario: Verify action toolbar elements
-    Given I open default "Table" component in noIFrame with "table" json from "commonComponents" using "selectable" object name
+    Given I open default "Table Test" component in noIFrame with "table" json from "commonComponents" using "selectable" object name
     When I check checkbox on header
     Then Action Toolbar elemens are visible and have "rgb(0, 129, 93)" color
```

Comment:
Slight spelling error on elements, think this should be changed as well as the table-steps files